packages/salesforcedx-vscode-apex-replay-debugger/package.json

```diff
@@ -28,6 +28,7 @@
     "@salesforce/core": "^3.23.2",
     "@salesforce/salesforcedx-apex-replay-debugger": "55.11.0",
     "@salesforce/salesforcedx-utils-vscode": "55.11.0",
+    "@salesforce/salesforcedx-utils": "55.11.0",
```

Comment:
Note this is the debugger adapter module so it pulling both vscode-utils and utils.

Reply:
How do we know which is the adapter module and not? is the -replay-debugger adapter module?

Reply:
sadly today by naming convention or by reading the config in the package.json files.  For us the salesforcedx-apex-debugger and salesforcedx-apex-replay-debugger are our debuggers and the salesforcedx-vscode-apex-debugger and salesforcedx-vscode-apex-replay-debugger are the adapters.

Reply:
`salesforcedx-vscode-apex-replay-debugger` and `salesforcedx-vscode-apex-debugger` are the adapter modules and `salesforcedx-apex-replay-debugger` and `salesforcedx-apex-debugger` are the debuggers referred in this PR.

Reply:
And the debuggers should call `salesforcedx-utils` and not the `salesforcedx-utils-vscode`